src/utils/apiResponse.utils.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
PullProofAI

Consider using TypeScript's built-in types instead of `any` for better type safety.
```diff
- export type ControllerResponse = {
+ export type ControllerResponse = {
-   data?: any;
+   data?: unknown;
-   error?: any;
+   error?: unknown;
```


<!-- This is an auto-generated comment by PullProofAI -->